templates/search_page.html

```diff
@@ -136,10 +137,19 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                     <div class="flex-fill">
                         <input class="form-control border-0" type="search" id="location" placeholder="Search"
-                            aria-label="Search" name="location" value="Los Angeles, CA, USA">
+                            aria-label="Search" name="location" value="Los Angeles, CA, USA" onchange="fetchLocation(this)"> 
                     </div>
 
-                    <div class="fixedwidthicon">
+                    <script>
```

Comment:
can we also move this JS script logic to its related main JS script? say `search.js`

Reply:
addressed in 6f483b0a